terraform/cloudfoundry/router_security_groups.tf

```diff
@@ -17,7 +17,7 @@ resource "aws_security_group" "web" {
     to_port   = 443
     protocol  = "tcp"
     cidr_blocks = [
-      "${split(",", var.web_access_cidrs)}",
+      "${compact(split("\n", replace(file("${path.root}/../${var.tenant_cidrs_file}"), "/ *#.*/", "")))}",
       "${var.concourse_elastic_ip}/32",
```

Comment:
We probably want to maintain ability to define different access levels to API vs WEB. E.g. in prod open web to all, but API only to tenants...